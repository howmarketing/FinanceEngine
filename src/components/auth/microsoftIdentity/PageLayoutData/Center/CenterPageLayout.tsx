import React from "react";

const Center: React.FC<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>> = (props) => {
    let styles: React.CSSProperties = props?.style || {};
    styles = (styles || {});
    styles = typeof styles === 'object' ? styles : {};
    styles = !(Array.isArray(styles)) ? styles : {};
    styles = { ...styles, display: 'flex', justifyContent: 'center', alignItems: 'center' };
    return (
        <div {...props} className="d-flex justify-content-center" style={styles}>
            {props?.children || ''}
        </div>
    )
}
export default Center;