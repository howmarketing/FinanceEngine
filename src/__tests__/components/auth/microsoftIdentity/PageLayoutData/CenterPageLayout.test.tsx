import React from 'react';
import TestRenderer from 'react-test-renderer';
import Center from 'pages/private/container/dashboard/DashboardComponent';

it('Page layout center component render', () => {
	const tree = TestRenderer.create(<Center />).toJSON()
	expect(tree).toMatchSnapshot();
})
