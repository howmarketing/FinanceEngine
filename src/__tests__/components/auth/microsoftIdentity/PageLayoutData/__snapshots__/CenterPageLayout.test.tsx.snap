// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page layout center component render 1`] = `
Array [
  <div
    className="header-container"
  >
    <h1
      className="container-title"
    >
      Dashboard
    </h1>
  </div>,
  " ",
]
`;
